five-phases/README.md

```diff
@@ -0,0 +1,26 @@
+Design Thinking is User-Centric.  Therefore the most important characteristic of a solution is that it meets the needs of the User.  To ensure solutions achieve this objective, the three fundamental principles of Design Thinking guide teams through the process of defining the MVP, and then iteratively improving and enhancing it to achieve the final product.
+
+These three principles are:
+1. Empathy
+2. Ideation
+3. Experimentation
+
+**Empathy** is simply understanding the needs of the user.  A designer must set aside their own assumptions about the problem so that they might gain this insight.  In Design Thinking, the team must always remain focused on the people for whom they are designing.
+
+**Ideation**  This principle centers on the generation of ideas - both many and diverse.  This principle allows for maximum creativity and innovation.
+
+**Experimentation** is also described as "launch and learn".  A Mimimum Viable Product is released and then iteratively developed with continuous feedback from users and other stakeholders. This approach aligns particularly well with Agile development practices.
+
+These three principles translate into five phases that result in an innovative and meaningful solution.  The five phases of Design Thinking are:
+1. Empathize - the development of a deep empathetic understanding of user needs.
+2. Define - using the knowledge gathered in phase one to specifically define the underlying problem to be solved.
```

Comment:
Use collected data to solving problems, based on the information, providing a solution